modules/social_features/social_post/modules/social_post_photo/src/Plugin/Block/PostPhotoBlock.php

```diff
@@ -23,12 +25,14 @@ class PostPhotoBlock extends PostBlock {
    */
   public function __construct(
     array $configuration,
-    $plugin_id,
-    $plugin_definition,
+          $plugin_id,
+          $plugin_definition,
```

Comment:
Although this is a common IDE setting, I'd expect PHPCS to complain about this change 🤔